dir.targets

```diff
@@ -1,120 +1,20 @@
-
-  <Target Name="_RestoreBuildTools"
-          Inputs="$(MSBuildThisFileDirectory)dir.props;$(SourceDir).nuget/packages.$(OsEnvironment).config"
-          Outputs="$(BuildToolsSemaphore)">
-    <Message Importance="High" Text="Restoring build tools..." />
-
-    <Copy Condition="Exists('$(NuGetCachedPath)')" SourceFiles="$(NuGetCachedPath)" DestinationFiles="$(NuGetToolPath)" SkipUnchangedFiles="true" />
-
-    <!-- Download latest nuget.exe -->
-    <DownloadFile FileName="$(NuGetToolPath)"
-                  Address="https://www.nuget.org/nuget.exe"
-                  Condition="!Exists('$(NuGetToolPath)') and '$(OsEnvironment)'=='Windows_NT'" />
-
-    <Exec Command="curl -sSL --create-dirs -o $(NuGetToolPath) https://api.nuget.org/downloads/nuget.exe"
-          Condition="!Exists('$(NuGetToolPath)') and '$(OsEnvironment)'=='Unix'" />
-
-    <PropertyGroup>
-      <_RestoreBuildToolsCommand>$(NugetRestoreCommand) "$(SourceDir).nuget/packages.$(OsEnvironment).config"</_RestoreBuildToolsCommand>
-    </PropertyGroup>
-
-    <!-- Restore build tools -->
-    <Exec Command="$(_RestoreBuildToolsCommand)" StandardOutputImportance="Low" />
-
-    <!-- Add DNU and Roslyn tool execute rights -->
-    <Exec Condition="'$(OsEnvironment)'=='Unix'"
-          Command="chmod a+x &quot;$(DnxPackageDir)/bin/dnu&quot;" />
-    <Exec Condition="'$(OsEnvironment)'=='Unix'"
-          Command="chmod a+x &quot;$(DnxPackageDir)/bin/dnx&quot;" />
-    <Exec Condition="'$(OsEnvironment)'=='Unix'"
-          Command="find '$(RoslynPackageDir)tools' -name &quot;*.exe&quot; -exec chmod &quot;+x&quot; '{}' ';'" />
-
-    <!--
-      Touch our semaphore file to ensure Inputs/Outputs comparison for this target will show that we're up to date.
-      Ignore failures in the unlikely, but possible, event that we hit this from two projects simultaneously.
-    -->
-    <Touch Files="$(BuildToolsSemaphore)"
-           ContinueOnError="WarnAndContinue"
-           AlwaysCreate="true"
-           ForceTouch="true" />
-
-    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true'"
-           Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
+  <Target Name="CheckForBuildTools">
+    <Error Condition="!Exists('$(ToolsDir)') and '$(OverrideToolsDir)'=='true'"
+           Text="The tools directory [$(ToolsDir)] does not exist. Please run sync in your enlistment to ensure the tools are installed before attempting to build an individual project." />
+    <Error Condition="!Exists('$(ToolsDir)') and '$(OverrideToolsDir)'!='true'"
+           Text="The tools directory [$(ToolsDir)] does not exist. Please run init-tools.cmd in your enlistment to ensure the tools are installed before attempting to build an individual project." />
```

Comment:
They aren't quite identical conditions. In the second clause one is == and other  !=